@here/harp-map-theme/resources/berlin_tilezen_night_reduced.json

```diff
@@ -620,26 +600,6 @@
                             2
                         ]
                     },
-                    "secondaryWidth": {
```

Comment:
Same as above.

Reply:
This properties are from solid-line technique, not dashed-line, maybe left from copy-paste